test.py

```diff
@@ -0,0 +1,157 @@
+import json
+import logging
+import types
+import webapp2
+from datetime import datetime, timedelta
+
+import persistentcache
+from google.appengine.api import memcache
+from google.appengine.ext import db
+
+
+TEST1_KEY = "test1"
+TEST2_KEY = "test2"
+
+# 2 megabyte payload
+# (uncomment if you can figure out how to work around appengine's inability to store huge entities)
```

Comment:
the usual trick is to split across records, with a key to help merge them, then a getter method to merge them together.  I have code for this if you need.